.coveragerc

```diff
@@ -1,7 +1,10 @@
 # .coveragerc to control coverage.py
 [run]
+branch = True
 omit =
-    src/scenic/simulators/webots/WBT*.py
+    tests/*
+    src/scenic/simulators/*
```

Comment:
Actually let's include `src/scenic/simulators/newtonian` and `src/scenic/simulators/utils` (all other submodules of `scenic.simulators` can be excluded though).